catalyst-service/src/main/resources/application.yml

```diff
@@ -21,7 +21,7 @@ spring:
           enabled: true
 
 # Temporary fix till camel 2.22 with spring boot 2 support
-camel.version: 2.20.2
+camel.version: 2.22.0
```

Comment:
This should be not needed anymore, as correct Camel version will be resolved by BOM